docs/metadata/Git/git-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Git Troubleshooting
+id: git-troubleshooting
+description: How to troubleshoot permissions and network issues with Git
+tags: []
+---
+
+Sometimes there is an error when Prophecy tries to reach Git. This could be caused by your user's [authorization](#debugging-git-credentials), a [repository setting](#debugging-repository-setttings), or a [network](#debugging-network-issues) issue. Fear not! This guide outlines what to do if you see error messages such as:  
+`not authorized`  
+`git-upload-pack-not-permitted`  
+or  
+`git-recieve-pack-not-permitted`
+
+![errorMsgs](./../img/git_troubleshooting_1.png)
+
+## Debugging Git Credentials
+
+Sometimes your personal access token (PAT) has expired or lacks a certain permission.
+
+1. Check if your PAT has expired. If so, create a new one with sufficient read and write access to the repository of interest. Here are the instructions to locate existing tokens and create new tokens for [Github](https://docs.github.com/en/enterprise-cloud@latest/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/repository-access-tokens/), or [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token).
+2. Open a terminal from a computer that can access your Git repository. If you need help with this step, please reach out to your Git administrator.
+3. Have the new PAT ready; you'll use it in place of a password.
+4. Check for read access by testing your ability to [clone](https://git-scm.com/docs/git-clone) a repo  
+   `git clone`  
+   Here are the clone instructions for [Github](https://docs.github.com/en/enterprise-cloud@latest/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#using-a-personal-access-token-on-the-command-line), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/clone-a-git-repository/), or [GitLab](https://docs.gitlab.com/ee/topics/git/clone.html).
+5. Check for write access by testing your ability to [push](https://git-scm.com/docs/git-push) to a repo  
+   `git push`  
+   Here are the push instructions for [Github](https://docs.github.com/en/enterprise-cloud@latest/get-started/using-git/pushing-commits-to-a-remote-repository), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/push-code-to-bitbucket/), or [GitLab](https://docs.gitlab.com/ee/topics/git/commands.html#git-push).
+6. If your PAT lacks read or write permissions for the repository of interest, ask the repository admin to adjust these permissions.
+7. If access has changed recently, check with your Git administrator to see if they have made any changes.
+
+## Debugging Repository Settings
+
+Sometimes a repository settings can restrict a user's permissions outside the user's profile settings page.
```

Comment:
I'm not sure where the reader would go to check the following restrictions. We could add a line saying something like, "On the repository's security settings, check for the following restrictions."